sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java

```diff
@@ -281,10 +288,29 @@ public PersonDirectorySteps(
           webDriverHelpers.selectFromCombobox(PRESENT_CONDITION_FILTER_COMBOBOX, "Alive");
         });
 
+    When(
+        "I check if Date of dead for specified case is correct",
+        () -> {
+          String date = webDriverHelpers.getValueFromWebElement(DATE_OF_DEATH_INPUT);
+          LocalDate deadDate = LocalDate.now().minusDays(1);
+          softly.assertEquals(DATE_FORMATTER.format(deadDate), date);
+          softly.assertAll();
+        });
+
+    When(
+        "I check if Cause of death is ([^\"]*)",
```

Comment:
I don't understand, this method checks if the filter was applied in combobox?

Reply:
this method checks if the correct cause of death is saved into filed causeOfDeath in edit person